compiler/noirc_evaluator/src/ssa/interpreter/value.rs

```diff
@@ -39,7 +39,7 @@ pub(crate) enum NumericValue {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub(crate) struct ReferenceValue {
+pub struct ReferenceValue {
```

Comment:
If you're going to automatically compare values across iterations, you should ignore the `original_id` field here since that may change from optimization passes

Reply:
Thanks! I only plan to compare the final return value, so I don't expect it to contain references, but good to have that insight 🙏